tensorflow/core/kernels/sparse_tensor_dense_matmul_op_gpu.cu.cc

```diff
@@ -46,7 +46,7 @@ __global__ void SparseTensorDenseMatMulKernel(int nnz, int m, int b_rows,
     // out[i, j]
     T* out_location = out + i * p + j;
     if (!FastBoundsCheck(k, n)) {
-      CudaAtomicAdd(out_location, std::numeric_limits<T>::quiet_NaN());
+      GpuAtomicAdd(out_location, Eigen::NumTraits<T>::quiet_NaN());
```

Comment:
Do we need to change this from STL to Eigen?

Reply:
I do not think we do....spotted the same after I did the rebase, and am currently testing out the change in our fork. will push out the change + rebase, once done